src/pages/AddListingPage/components/StepIndicator.tsx

```
@@ -0,0 +1,60 @@
+                  step.id < currentStep
+                    ? "border-[#2563eb] bg-[#2563eb]"
+                    : "border-[#2563eb] bg-white"
+                }`}
+              >
+                {step.id < currentStep ? (
+                  <svg
+                    className="size-4 text-white"
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    stroke="currentColor"
+                  >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)